common/mount.py

```diff
@@ -592,13 +616,14 @@ def _umount(self):
             exceptions.MountException: If unmount failed.
         """
         try:
-            subprocess.check_call(['fusermount', '-u', self.currentMountpoint])
+            subprocess.check_call(["fusermount", "-u", self.currentMountpoint])
 
         except subprocess.CalledProcessError as exc:
             raise MountException(
-                _("Can't unmount {mountprocess} from {mountpoint}.")
-                .format(mountprocess=self.mountproc,
-                        mountpoint=self.currentMountpoint)) from exc
+                _("Can't unmount {mountprocess} from {mountpoint}.").format(
```

Comment:
Here `"` is allowed because `'` is contained in the content.
I also updated the contribution info about it. PR #1804
My apologize for not mention that earlier. I am aware that it is not obvious from looking into the code base because there it is mixed.